development-defaults.ini

```diff
@@ -1,5 +1,5 @@
 
-path = "/uber"
```

Comment:
Whoops, this again.

I'd really like to solve this since it makes merging fixes from upstream a pain. Is it maybe possible to put this in puppet for MAGFest? @binary1230 what do you think?